### PR TITLE
add three data augmentation apis（3-Augment， Augmix，TrivialAugmentWide）

### PR DESCRIPTION
**Pull request by sageyou**
Thank you for your contribution to the MindCV repo.
Before submitting this PR, please make sure:

- [ ] You have read the [Contributing Guidelines on pull requests](https://github.com/mindspore-lab/mindcv/blob/main/CONTRIBUTING.md)
- [ ] Your code builds clean without any errors or warnings
- [ ] You are using approved terminology
- [ ] You have added unit tests

## Motivation

(Write your motivation for proposed changes here.)

## Test Plan

(How should this PR be tested? Do you require special setup to run the test or repro the fixed bug?)

## Related Issues and PRs

(Is this PR part of a group of changes? Link the other relevant PRs and Issues here. Use https://help.github.com/en/articles/closing-issues-using-keywords for help on GitHub syntax)

### CODE REVIEW

**Review by geniuspatrick — APPROVED** [review]

File: mindcv/data/transforms_factory.py [hunk 1]
```diff
@@ -65,8 +70,13 @@ def transforms_imagenet_train(
         augement_params["interpolation"] = interpolation
         if auto_augment.startswith("randaug"):
             trans_list += [rand_augment_transform(auto_augment, augement_params)]
-        elif auto_augment.startswith("autoaug") or auto_augment.startswith("autoaugr"):
+        elif auto_augment.startswith("autoaug") or auto_augment.startswith("autoaugr") or auto_augment.startswith("3a"):
```
Comment (SamitHuang) [inline mindcv/data/transforms_factory.py:73]: 这个条件判断跟可精简，且与71合并。

File: tests/modules/test_transforms.py [hunk 2]
```diff
@@ -17,7 +17,8 @@
 @pytest.mark.parametrize("name", ["ImageNet"])
 @pytest.mark.parametrize("image_resize", [224, 256])
 @pytest.mark.parametrize("is_training", [True, False])
-def test_transforms_standalone_imagenet(mode, name, image_resize, is_training):
+@pytest.mark.parametrize("auto_augment", [None, "3a", "augmix", "trivialaugwide"])
```
Comment (SamitHuang) [inline tests/modules/test_transforms.py:20]: include autoaug, randaug?

Comment (sageyou) [reply tests/modules/test_transforms.py]: ok， will add this two also~

File: mindcv/data/transforms_factory.py (hunk 1 above)
Comment (sageyou) [reply mindcv/data/transforms_factory.py]: 我把这行精简了， 然后和71行合并的话？ 这是两个不同的接口诶（74 和72 调用的）

**Review by SamitHuang — APPROVED** [review]

**Pull request merged**